src/Awaitable.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Interop\Async;
+
+/**
+ * Awaitable object representing the future value of an asynchronous operation.
```

Comment:
Next suggestion: `Representation of a future value of an asynchronous operation.`